src/library/basetracktablemodel.cpp

```diff
@@ -35,6 +35,7 @@ const mixxx::Logger kLogger("BaseTrackTableModel");
 constexpr double kRelativeHeightOfCoverartToolTip =
         0.165; // Height of the image for the cover art tooltip (Relative to the available screen size)
 
+// Mandatory columns that are either used directly or in delegates.
 const QStringList kDefaultTableColumns = {
```

Comment:
unrelated to your changes, just something I noticed in general: The content here doesn't actually matter, the only thing thats ever used by `kDefaultTableColumns` is its size. Thats probably not how this was supposed to be used originally.

Reply:
It is used if you derive your custom model from `BaseTrackTableModel` and want/need to reuse the implicitly defined and supported default columns.

`BaseSqlTableModel` seems to compose its own list that somehow matches the implicit requirements of its base class.

The whole design is neither composable nor maintainable. I just use it as is.

Reply:
Good catch: https://github.com/mixxxdj/mixxx/pull/13682/commits/9fdafecad0ad55ce15621703a16564ee00954e06

I'll maintain a local patch and need to keep track of all future changes and extensions myself.

Reply:
Thanks for doing the cleanup (even though I didn't ask you to).

Reply:
It was probably me who added this unneeded code when splitting off the base class  :sweat_smile: